### PR TITLE
some draft edit suggestions

### PR DESCRIPTION
**Pull request by kranzky**
i made edits to your draft as i read it; make of them what you will. also:

* `configure.sh` couldn't find node; I use `nvm` and it was clearly installed in `/Users/jason/.nvm/versions/node/v7.4.0/bin/node`
* `hello.bin` is not in the repository
* you might want to create a `.gitignore` and add `bin` and `disasm`

see the output here: https://github.com/jasonhutchens/mipsdis/blob/master/draft.md

### CODE REVIEW

File: draft.md [hunk 1]
```diff
@@ -1,60 +1,56 @@
 # Mipsdis
 
-`mipsdis` is a MIPS bytecode disassembler that I've been hacking on that uses source code generation. For those in a hurry, [here](http://blog.loadzero.com/draft/mipsdis.html) is a link to the obligatory in-browser demo.
+`mipsdis` is a MIPS bytecode disassembler that uses source code generation.
```
Comment (kranzky) [inline draft.md:3]: suggest the first line is a very brief overview

File: draft.md [hunk 2]
```diff
@@ -1,60 +1,56 @@
...
 
-I wrote `mipsdis` because I'm interested in low-level software and getting to
-grips with an [ISA](https://en.wikipedia.org/wiki/Instruction_set) has been on
-my list for a while. As far as that goes, I thought writing a
-[disassembler](https://en.wikipedia.org/wiki/Disassembler) would be a good
-first step.
+[Live Demo](http://blog.loadzero.com/draft/mipsdis.html)
```
Comment (kranzky) [inline draft.md:5]: ^ get people playing with the demo up-front I reckon

File: draft.md [hunk 3]
```diff
@@ -1,60 +1,56 @@
...
-writing an [x86](https://en.wikipedia.org/wiki/X86) disassembler would take me
-about ten times as long. I am unsure about
-[ARM](https://en.wikipedia.org/wiki/ARM_architecture), but I get the feeling
-that it is more complex than MIPS.
-
-I decided to use source code generation for a few reasons.
+This article will demonstrate the different versions of the disassembler,
+discuss the code generator, describe how to extend `mipsdis`, provide
+instructions for getting it all running yourself, and conclude by listing
+some of the extra documentation available for the project.
```
Comment (kranzky) [inline draft.md:10]: ^ frontload this overview; sets the scene

File: draft.md [hunk 4]
```diff
@@ -1,60 +1,56 @@
...
 
-Firstly, I find writing a bunch of slightly different dispatch and decoding
-routines by hand is tedious and error-prone. I find manipulating metadata using
-a simple and commented text format to be much more pleasant.
+## Background
```
Comment (kranzky) [inline draft.md:12]: ^ suggest sub-sections

**Comment by loadzero** [comment]
Cheers. Merging manually.

**Pull request closed without merging**